deploying-daml/daml-on-fabric/daml_deploy_app.md

```diff
@@ -0,0 +1,6 @@
+Alright, now to deploy our application to our DAML Runtime using our ever-handy `daml` assistant.
+
```

Comment:
The process is done when you see the `DAR upload succeeded.
Deploy succeeded.` message in the terminal.